requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Use >= for development versions so that source builds always work
-coala>=0.8.0.dev20160718123552
+coala>=0.8.0.dev20160729111621
```

Comment:
It picks up https://github.com/coala-analyzer/coala/commit/6d96e937d3e026917a6ce2f02a8a9a06698531e4